debezium-core/src/main/java/io/debezium/pipeline/source/snapshot/incremental/AbstractIncrementalSnapshotChangeEventSource.java

```diff
@@ -176,7 +176,7 @@ protected void sendEvent(P partition, EventDispatcher<P, T> dispatcher, OffsetCo
      */
     protected ChangeRecordEmitter<P> getChangeRecordEmitter(P partition, T dataCollectionId,
                                                             OffsetContext offsetContext, Object[] row) {
-        return new SnapshotChangeRecordEmitter<>(partition, offsetContext, row, clock);
+        return new SnapshotChangeRecordEmitter<>(partition, offsetContext, row, clock, connectorConfig);
```

Comment:
Not really necessary here, should I create a different constructor instead? @Naros @jpechane

Reply:
I think it's OK to add the constructor argument that way, but I made a suggestion below regarding how far up the class hierarchy this should be passed.

Reply:
Ok